src/lib/index.jsx

```diff
@@ -24,6 +29,7 @@ type Props = {
   isInputNum?: boolean,
   value?: string,
   className?: string,
+  onKeyDown?: KeyDownProp
```

Comment:
It'd be conventional to take a regular function here like other react `onKeyDown` handlers.